jetty-websocket/websocket-jetty-tests/src/test/java/org/eclipse/jetty/websocket/tests/client/ClientConfigTest.java

```diff
@@ -58,13 +58,12 @@
     private Server server;
     private WebSocketClient client;
     private ServerConnector connector;
+    private final EchoSocket serverSocket = new EchoSocket();
 
-    private EchoSocket serverSocket = new EchoSocket();
-
-    private static String message = "this message is over 20 characters long";
-    private final int inputBufferSize = 200;
-    private final int maxMessageSize = 20;
-    private final int idleTimeout = 500;
+    private static final String message = "this message is over 20 characters long";
```

Comment:
static finals are uppercase, underscore separated, according to our coding conventions. Eg `INPUT_BUFFER_SIZE`